packages/drivers/iframe-driver/src/debug.ts

```diff
@@ -4,7 +4,18 @@
  */
 
 import registerDebug from "debug";
+import { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/common-definitions";
 import { pkgName, pkgVersion } from "./packageVersion";
 
 export const debug = registerDebug("fluid:iframe-socket-storage");
 debug(`Package: ${pkgName} - Version: ${pkgVersion}`);
+
+export class IFrameDebugLogger implements ITelemetryBaseLogger {
```

Comment:
we already have a DebugLogger. we should just use that

Reply:
i think this whole file can be deleted

---
In reply to: [513620197](https://github.com/microsoft/FluidFramework/pull/4106#discussion_r513620197) [](ancestors = 513620197)

Reply:
the debug itself is still being used in a few places, so going to hold off on removing that for now